Themes/default/Errors.template.php

```diff
@@ -69,31 +69,43 @@ function template_error_log()
 					<a href="', $scripturl, '?action=helpadmin;help=error_log" onclick="return reqOverlayDiv(this.href);" class="help"><span class="main_icons help" title="', $txt['help'], '"></span></a> ', $txt['errorlog'], '
 				</h3>
 			</div>
-			<div class="pagesection">
-				<div class="floatleft">
-					', $context['page_index'], '
-				</div>';
+			<div class="information flow_hidden">
+				<div class="additional_row">';
 
-	if (!empty($context['errors']))
+	// No errors, so just show a message and be done with it.
+	if (empty($context['errors']))
+	{
 		echo '
+					', $txt['errorlog_no_entries'], '
+				</div>
+			</div>
+		</form>';
+		return;
+	}
+
+	echo '
 				<div class="floatright">
 					<input type="submit" name="removeSelection" value="', $txt['remove_selection'], '" data-confirm="', $txt['remove_selection_confirm'], '" class="button you_sure">
 					<input type="submit" name="delall" value="', ($context['has_filter'] ? $txt['remove_filtered_results'] : $txt['remove_all']), '" data-confirm="', ($context['has_filter'] ? $txt['remove_filtered_results_confirm'] : $txt['sure_about_errorlog_remove']), '" class="button you_sure">
-				</div>';
-
-	echo '
-			</div>
-			<div class="title_bar">
-				<div style="padding: 6px 12px 5px 12px">
-					', $txt['apply_filter_of_type'], ':';
+				</div>
+				', $txt['apply_filter_of_type'], ':';
```

Comment:
There's something strange in this area, the indentation doesn't look right?

Reply:
It just looks weird because lines 75-84 above quickly close the wrapper divs and return when there are no errors.